site/config.toml

```diff
@@ -34,9 +34,9 @@ pygmentsStyle                 = "manni"
 [params]
     description               = "We're a community of Latinx professionals in the tech industry. 🌮️"
     copyright                 = "Techqueria.org"
-    year                      = 2017
-    og_image                  = "https://i.imgur.com/A43r74s.png"
-    logo                      = "https://i.imgur.com/mLO2a2t.png"
+    year                      = 2016
+    og_image                  = "https://i.imgur.com/QZAbJUJ.png"
```

Comment:
The banner spacing seems to be off a little - would it be possible to add some spacing below the logo and aligning both to be in the center?

![New Banner](https://i.imgur.com/QZAbJUJ.png)

Otherwise, this looks great!

Reply:
@dvidsilva Any update on the spacing issue?